pkg/network/ovn_kubernetes.go

```diff
@@ -123,6 +123,11 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["RoutableMTU"] = nil
 	// v4 and v6 join subnet are used when the user wants to use the addresses that we reserve for the join subnet in ovn-k
 	// TODO: this field is being deprecated and will turn into c.GatewayConfig.IPv4/6.InternalJoinSubnet when we introduce the transit switch config into the api
+	data.Data["V4JoinSubnet"] = ""
+	data.Data["V6JoinSubnet"] = ""
+	data.Data["V4TransitSwitchSubnet"] = ""
+	data.Data["V6TransitSwitchSubnet"] = ""
+
```

Comment:
curious, why do we need this initialization?

Reply:
If we dont initialize and c.IPv4.InternalTransitSwitchSubnet is "" then V4TransitSwitchSubnet does not get set to any value. as a result, RenderTemplate() starts throughing variable undefined error as I am using V4TransitSwitchSubnet in 008.script-lib.yaml. 
Same will also happen for join subnet when we would deprecate c.V4InternalSubnet.